fluentlenium-core/src/test/java/org/fluentlenium/integration/FluentLeniumWaitTest.java

```diff
@@ -152,37 +182,37 @@ public void checkAwaitContainsTextWithTextMatcher() {
 
     @Test
     public void when_a_element_is_not_present_then_isNotPresent_return_true() {
-        await().atMost(1, NANOSECONDS).until(".small").withText().contains("notPresent").isNotPresent();
+        await().atMost(1, NANOSECONDS).until(".small").withText().contains("notPresent").not().isPresent();
```

Comment:
I'm not sure `isPresent` is still good mayeb it would be better to replace it with just `present` ? It would be easier to read. Not is present reads strange to me.